ci/single_binary_test.d

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env rdmd
+module system_integration_test;
+
+private:
+
+import std.algorithm;
+import std.file;
+import std.format;
+import std.path;
+import std.process;
+import std.stdio;
+import core.thread;
+
+/// Root of the repository
+immutable RootPath = __FILE_FULL_PATH__.dirName.dirName;
+immutable IntegrationPath = RootPath.buildPath("tests").buildPath("system");
+immutable LauncherPath = RootPath.buildPath("tests").buildPath("launcher");
+
+/+ ***************************** Commands to run **************************** +/
+/// A simple test to ensure that the container works correctly,
+/// e.g. all dependencies are installed and the binary isn't corrupt.
+immutable BuildBinary = [ "dub", "build", "-c", "launcher", ];
+immutable BinaryUp = [ "build/agora-launcher", "-c", LauncherPath.buildPath("configs.yaml"), ];
+immutable BinaryDown = [ "killall", "-9", "agora-launcher", ];
```

Comment:
That is not an option. We should make sure we got a clean finish

Reply:
I will add a command API to agora-launcher.

Reply:
I solved it with ctrl+c.